### PR TITLE
Convert audience-item.js to es6

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Converts audience-item.js to es6

## What is the value of this and can you measure success?

More es6!

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

## Tested in CODE?

No

### CODE REVIEW

File: static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import bonzo from 'bonzo';
 import debounce from 'lodash/functions/debounce';
 
 class ABTestReportItem extends Component {
-    constructor(config: any): void {
+    constructor(config: Object): void {
```
Comment (GHaberis) [inline static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js]: Not strictly part of this es6 conversion but 'Object' is the correct type of config.

Comment (lps88) [inline static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js]: I have so far only seen `(config: {})` style used for object types. I'm interested if there is a difference? 🤔

**Review by SiAdcock — APPROVED** [review]
Excellent

File: static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js (hunk 1 above)
Comment (GHaberis) [inline static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js]: Interesting, I've not actually seen an empty object literal used as the type placeholder yet, only object literals where the expected properties were also defined. I'll look into the difference.

**Pull request merged**